auth_password_settings/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+User Password Settings
+======================
+
+This module allows an admin to manage Password Strength rules, which would be enforced on next password change.
+
+Installation
+============
+
+To install this module, you need to:
+
+* Nothing special
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Go to Settings->Configuration->General Settings and setup the Password Strength section
+
+ * Minimum Password length in character
+ * Use Capital Letter
+ * Use digit
+ * Use special character
+
+
+Usage
+=====
+
+This module will enforce the selected rules on password change.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/149/8.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/server_tools/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/server-tools/issues/new?body=module:%20auth_password_settings%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
```

Comment:
The icon is from @dhp-denero original PR #134 - I don't know where he got it
Probaly better to replace it with the OCA icon?